### PR TITLE
Build sustainability page

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built sustainability page as per [design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=7390%3A21105&t=yLxIeHDfRahCjLgS-0) and [copy doc](https://docs.google.com/document/d/1wwSOXczrGS45e1DiXbt4GMsHKolniRlywxV_CNHHdCA/)
- Upgrade to latest vanilla version (needed to fix list text wrapping issue) 
 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-809.demos.haus/careers/sustainability
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against design and copy doc 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1818

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@mtruj013 is this using the newly released divided list? this kind of wrapping is hopefully fixed there: 
image

**Comment by lyubomir-popov** [comment]
please insert a break as in the design, this is a little dense 
image

**Comment by lyubomir-popov** [comment]
Can we maintain the design through the medium breakpoint:
image

**Comment by lyubomir-popov** [comment]
@mtruj013 what is blocking this?

**Comment by mtruj013** [comment]
> @mtruj013 what is blocking this?

@lyubomir-popov Originally, the missing image (see PR description), however I spoke with Gloria recently who communicated a way larger blocker. I'll leave the context in the careers channel

**Comment by marinacastejon** [comment]
@mtruj013 Thanks, UX-wise LGTM! The visual design of the top section doesn't seem consistent with this [mock-up](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=7390-21031&t=62mO0sz5PNOPVDBj-0), but maybe you and @lyubomir-popov discussed that separately.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
@mtruj013 this is looking good. Just some comments and suggestions in the code. Maybe also add the page to the lifestyle tests.

File: templates/careers/sustainability.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %} https://docs.google.com/document/d/1wwSOXczrGS45e1DiXbt4GMsHKolniRlywxV_CNHHdCA/edit {% endblock meta_copydoc %}
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/careers/sustainability.html:3]: Ah I did wonder if that was why but thought I'd mention it. If the browser plugins still works with it then that's fine.

**Review by anthonydillon — APPROVED** [review]
Code and QA LGTM :+1:

**Comment by tbille** [comment]
@lyubomir-popov can you take a look?

File: webapp/app.py [hunk 2]
```diff
@@ -319,7 +319,8 @@ def all_careers():
 
 
 @app.route("/careers/travel")
-def working_here_travel_page():
+@app.route("/careers/sustainability")
+def working_here_pages():
```
Comment (carkod) [inline webapp/app.py:323]: Can we add tests for these please? Thanks.

Comment (mtruj013) [reply webapp/app.py:323]: Done!

**Comment by mtruj013** [comment]
Thanks @lyubomir-popov! I've pushed the changes if you could take a look again?

**Comment by anthonydillon** [comment]
Recent changes seem good too :+1:

**Pull request merged**